backend/src/main/java/it/chalmers/gamma/controller/administrationSubControllers/AuthorityAdminController.java

```diff
@@ -0,0 +1,125 @@
+package it.chalmers.gamma.controller.administrationSubControllers;
+
+import com.sun.mail.iap.Response;
+import it.chalmers.gamma.db.entity.*;
+import it.chalmers.gamma.db.serializers.ITUserSerializer;
+import it.chalmers.gamma.requests.AuthorizationLevelRequest;
+import it.chalmers.gamma.requests.AuthorizationRequest;
+import it.chalmers.gamma.response.*;
+import it.chalmers.gamma.service.*;
+import org.json.simple.JSONObject;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.Null;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/admin/authorization")
+public class AuthorityAdminController {
```

Comment:
Should be named `AuthorizationAdminController` to match the request mapping

Reply:
No

Reply:
Can we call it either the one or the other

Reply:
I'd rather change the endpoint than the class name. Makes more sense internally

Reply:
Do it 👍